ext/nnstreamer/extra/meson.build

```diff
@@ -32,8 +32,12 @@ if grpc_support_is_available
     error('failed to resolve dependency: gRPC requires libprotobuf.')
   endif
 
-  # Don't generate .proto file twice
-  nns_protobuf_grpc_lib = shared_library ('nnstreamer_protobuf_grpc',
+  if not flatbuf_support_is_available
+    error('failed to resolve dependency: gRPC requires libflatbuf.')
+  endif
```

Comment:
Do you really need to make gRPC require both protobuf & flatbuf?
Or would it be possible to require at least one of the two?

Reply:
You're right. Although gRPC uses protobuf by default, it does not require any mandatory IDL.
But, in this design, each tensor_src/sink can choose IDL from a property, so I decided to check both dependencies.

If you think it does not make sense, I will try to distinguish dependencies.

Reply:
Please write an issue of this. Let's do it later.